index.html

```diff
@@ -1,17 +1,88 @@
+<!DOCTYPE html>
 <html>
 	<head>
 		<meta charset="utf-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<title>Cookie Counter</title>
 		<link rel="stylesheet" type="text/css" href="css/reset.css"/>
-		<link rel="stylesheet" type="text/css" href="css/style.css"/>
+    <link rel="stylesheet" type="text/css" href="css/style.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Emilys+Candy|Kranky|Lily+Script+One|Oleo+Script+Swash+Caps|Vast+Shadow" rel="stylesheet">
```

Comment:
It looks like you are importing quite a few fonts, but only using one. You should only import the fonts you are using because more fonts means more time to load your page.

Reply:
Also, you need to have this `link` tag on the `sales.html` page as well if you want to make use of any of these fonts on that page.